jenkins/mysql/docker-compose.yml

```
@@ -15,8 +15,6 @@ services:
       MYSQL_DATABASE: ${ENV_LISK_DB_DATABASE}
       MYSQL_USER: ${ENV_LISK_DB_USER}
       MYSQL_PASSWORD: ${ENV_LISK_DB_PASSWORD}
-    expose:
-      - 3306
     ports:
       - "3306:3306"
```

Is there a reason not to bind to localhost only?

```suggestion
      - "127.0.0.1:3306:3306"
```